java/src/org/openqa/selenium/bidi/network/SetCacheBehaviorParameters.java

```diff
@@ -0,0 +1,48 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.bidi.network;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.openqa.selenium.internal.Require;
+
+public class SetCacheBehaviorParameters {
+  private final CacheBehavior cacheBehavior;
+  private final List<String> contexts;
+
+  public SetCacheBehaviorParameters(CacheBehavior cacheBehavior) {
+    this(cacheBehavior, null);
+  }
+
+  public SetCacheBehaviorParameters(CacheBehavior cacheBehavior, List<String> contexts) {
```

Comment:
If it is just these parameters, then we don't necessarily need a separate class for it. We can have overloaded methods in the Network class itself

Reply:
Ok, I will remove the `SetCacheBehaviorParameters` class. The new usage will be like:

```java
network.setCacheBehavior(
              CacheBehavior.BYPASS, Collections.singletonList(contextId));
```